src/object/bicycle_platform.cpp

```diff
@@ -156,58 +174,43 @@ BicyclePlatform::update(float dt_sec)
   m_angle += m_angular_speed * dt_sec;
   m_angle = math::positive_fmodf(m_angle, math::TAU);
 
-  m_angular_speed = std::min(std::max(m_angular_speed, -128.0f * math::PI * dt_sec),
-                             128.0f * math::PI * dt_sec);
+  m_angular_speed = std::min(std::max(m_angular_speed, -128.f * math::PI * dt_sec),
+                             128.f * math::PI * dt_sec);
 
-  if (m_walker)
-  {
-    m_walker->update(std::max(0.0f, dt_sec * m_angular_speed * 0.1f));
-    m_center = m_walker->get_pos(Sizef(), {});
-  }
-  else
-  {
-    m_center += Vector(m_angular_speed, 0) * dt_sec * 32.0f;
-  }
-}
+  get_walker()->m_walking_speed = m_angular_speed;
 
-void
-BicyclePlatform::on_flip(float height)
-{
-  m_center.y = height - m_center.y;
-}
-
-void
-BicyclePlatform::editor_delete()
-{
-  // Remove children.
-  for (auto& child : m_children)
-  {
-    child->remove_me();
-  }
+  std::cout << m_angular_speed << std::endl;
```

Comment:
Remove debug code (in other places too).

Reply:
This pull request is not ready for review

Reply:
You requested review from me.

Reply:
Oh, you're right, I did. It was an accident, sorry